Makefile

```diff
@@ -33,10 +33,10 @@ E2E_TEST_OPTIONS := -count=1 -v
 
 # useful for other docker repos
 REGISTRY ?= mcr.microsoft.com/k8s/aad-pod-identity
-NMI_IMAGE_NAME := $(REGISTRY)/$(NMI_BINARY_NAME)
-MIC_IMAGE_NAME := $(REGISTRY)/$(MIC_BINARY_NAME)
-DEMO_IMAGE_NAME := $(REGISTRY)/$(DEMO_BINARY_NAME)
-IDENTITY_VALIDATOR_IMAGE_NAME := $(REGISTRY)/$(IDENTITY_VALIDATOR_BINARY_NAME)
+NMI_IMAGE_NAME := $(REGISTRY)/$(NMI_BINARY_NAME):$(NMI_VERSION)
```

Comment:
Can you please change this part to make it possible to read the image names from environment ?